tests/functional/oauth_choose_redirect.js

```diff
@@ -23,7 +23,6 @@ define([
 
   registerSuite({
     name: 'oauth choose redirect',
-
```

Comment:
Can you revert the newline changes in this file to reduce the noise ever so slightly?